#: coalib/output/ConsoleOutputter.py:42
msgid "One of the given settings ({}) are not properly described."
msgstr ""

#. Translators: this is the and that connects the last two items of an enumeration (1st, 2nd AND 3rd)
#: coalib/output/ConsoleOutputter.py:49
#: coalib/tests/output/ConsoleOutputterTest.py:46
#: coalib/tests/output/ConsoleOutputterTest.py:56
msgid " and "
msgstr ""

#: coalib/output/ConsoleOutputter.py:51
#: coalib/tests/output/ConsoleOutputterTest.py:38
#: coalib/tests/output/ConsoleOutputterTest.py:43
#: coalib/tests/output/ConsoleOutputterTest.py:53
msgid "Please enter a value for the setting \"{}\" ({}) needed by {}: "
msgstr ""

#: coalib/output/LogPrinter.py:34 coalib/tests/output/LogPrinterTest.py:54
#: coalib/tests/processes/communication/LogMessageTest.py:50
#: coalib/processes/communication/LogMessage.py:31
msgid "DEBUG"
msgstr ""

#: coalib/output/LogPrinter.py:35 coalib/tests/output/LogPrinterTest.py:57
#: coalib/tests/processes/communication/LogMessageTest.py:48
#: coalib/processes/communication/LogMessage.py:32
msgid "WARNING"
msgstr ""

#: coalib/output/LogPrinter.py:36 coalib/tests/output/LogPrinterTest.py:47
#: coalib/tests/output/LogPrinterTest.py:50
#: coalib/tests/output/LogPrinterTest.py:60
#: coalib/tests/output/LogPrinterTest.py:63
#: coalib/tests/processes/communication/LogMessageTest.py:46
#: coalib/tests/processes/communication/LogMessageTest.py:52
#: coalib/processes/communication/LogMessage.py:33
msgid "ERROR"
msgstr ""

#: coalib/output/LogPrinter.py:55 coalib/tests/output/LogPrinterTest.py:64
msgid "Exception was:"
msgstr ""

#. pragma: no cover
#: coalib/output/EspeakPrinter.py:33
msgid ""
"Espeak doesn't seem to be installed. You cannot use the voice output feature "
"without espeak. It can be downloaded from http://espeak.sourceforge.net/ or "
"installed via your usual package repositories."
msgstr ""

#. pragma: no cover
#: coalib/output/EspeakPrinter.py:38
msgid "Failed to execute espeak. An unknown error occurred."
msgstr ""

#: coalib/misc/StringConstants.py:19
msgid ""
"This is a bug. We are sorry for the inconvenience. Please contact the "
"developers if this happens more than once."
msgstr ""

#: coalib/misc/StringConstants.py:22
msgid "{} is not accessible and will be ignored!"
msgstr ""

#: coalib/misc/StringConstants.py:30
msgid "on"
msgstr ""

#: coalib/misc/StringConstants.py:30
msgid "y"
msgstr ""

#: coalib/misc/StringConstants.py:30
msgid "yes"
msgstr ""

#: coalib/misc/StringConstants.py:30
#: coalib/tests/misc/StringConverterTest.py:64
msgid "yeah"
msgstr ""

#: coalib/misc/StringConstants.py:30
msgid "sure"
msgstr ""

#: coalib/misc/StringConstants.py:30
msgid "true"
msgstr ""

#: coalib/misc/StringConstants.py:31
msgid "definitely"
msgstr ""

#: coalib/misc/StringConstants.py:31
msgid "yup"
msgstr ""

#: coalib/misc/StringConstants.py:31
msgid "right"
msgstr ""

#: coalib/misc/StringConstants.py:39
msgid "off"
msgstr ""

#: coalib/misc/StringConstants.py:39
msgid "n"
msgstr ""

#: coalib/misc/StringConstants.py:39
msgid "no"
msgstr ""

#: coalib/misc/StringConstants.py:39
#: coalib/tests/misc/StringConverterTest.py:68
msgid "nope"
msgstr ""

#: coalib/misc/StringConstants.py:39
msgid "nah"
msgstr ""

#: coalib/misc/StringConstants.py:39
msgid "false"
msgstr ""

#: coalib/misc/StringConstants.py:39
msgid "wrong"
msgstr ""

#. Do not change this translation without changing it in the code also!
#: coalib/tests/misc/i18nTest.py:46 coalib/tests/misc/i18nTest.py:50
#: coalib/tests/bears/BearTest.py:41 coalib/tests/bears/BearTest.py:81
#: coala:25
msgid "A string to test translations."
msgstr ""

#: coalib/tests/bears/BearTest.py:78 coalib/tests/bears/BearTest.py:88
#: coalib/bears/Bear.py:98
msgid "Setting up bear {}..."
msgstr ""

#: coalib/tests/bears/BearTest.py:80 coalib/tests/bears/BearTest.py:89
#: coalib/bears/Bear.py:100
msgid "Running bear {}..."
msgstr ""

#: coalib/tests/bears/BearTest.py:82 coalib/tests/bears/BearTest.py:90
#: coalib/bears/Bear.py:114
msgid "Tearing down bear {}..."
msgstr ""

#. debug message contains custom content, dont test this here
#: coalib/tests/bears/BearTest.py:92 coalib/bears/Bear.py:105
#: coalib/bears/Bear.py:124
msgid "Bear {} failed to run"
msgstr ""

#. pragma: no cover
#: coalib/processes/BearRunner.py:113
msgid ""
"An unknown error occurred while running local bears for the file {}. "
"Skipping file..."
msgstr ""

#. pragma: no cover
#: coalib/processes/BearRunner.py:130
msgid ""
"An unknown error occurred while running global bear {}. Skipping bear..."
msgstr ""

#: coalib/processes/BearRunner.py:144
msgid "An internal error occurred."
msgstr ""

#: coalib/processes/BearRunner.py:145
msgid "The given file through the queue is not in the file dictionary."
msgstr ""

#: coalib/processes/BearRunner.py:159 coalib/processes/BearRunner.py:170
msgid "A given local bear ({}) is not valid. Leaving it out..."
msgstr ""

#: coalib/bears/Bear.py:106
#, python-brace-format
msgid ""
"The bear {bear} raised an exception of type {exception}. If you are the "
"writer of this bear, please catch all exceptions. If not and this error "
"annoys you, you might want to get in contact with the writer of this bear.\n"
"\n"
"Here is your traceback:\n"
"\n"
"{traceback}\n"
msgstr ""

#: coalib/bears/Bear.py:120
msgid ""
"set_up() or tear_down() throwed an exception for bear {}.\n"
"Exception: {}\n"
"Traceback:\n"
"{}"
msgstr ""

#: coalib/settings/SettingsFiller.py:61
msgid "One of the given bears ({}) has no attribute get_needed_settings."
msgstr ""

#. Something's gone wrong
#: coala:28
msgid "Program terminated by user."
msgstr ""
